docs/code_contribution_guidelines.md

```diff
@@ -464,10 +605,10 @@ Rejoice as you will now be listed as a [contributor](https://github.com/lightnin
 - [&nbsp;&nbsp;] For code and documentation: lines are wrapped at 80 characters
   (the tab character should be counted as 8 characters, not 4, as some IDEs do
   per default)
-- [&nbsp;&nbsp;] Running `go test` does not fail any tests
+- [&nbsp;&nbsp;] Running `make check` does not fail any tests
```

Comment:
Can this section be replaced by a link to https://github.com/lightningnetwork/lnd/blob/master/.github/pull_request_template.md ?

Reply:
Would rather have it inline so the document stands on its own.

Reply:
I wish it were possible to "embed" another markdown doc kinda like a template